.dialyzer_ignore.exs

```diff
@@ -0,0 +1,4 @@
+[
+  {"lib/mix/tasks/siwapp.ex"},
+  {"lib/mix/tasks/siwapp.register.ex"}
+]
```

Comment:
could this be a pattern for the whole directory? Something like "lib/mix/tasks/*"

Reply:
I've pushed a new commit with that. "lib/mix/tasks/*" didn't work, I was only able to do it with reg. exp: ~r/lib\/mix/

Reply:
Great choice, I think we found the correct solution. Hard to deal with RegEx in Elixir, right?

Reply:
Haha yes!! Thank you for helping me :)